### ISSUE TITLE
Crash report 7.2: NPE in PostsListAdapter

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#5796]
Check already reported/fixed https://github.com/wordpress-mobile/WordPress-Android/issues/5539

````
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.PostsListActivity}: android.view.InflateException: Binary XML file line #11: Binary XML file line #11: Error inflating class fragment
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2947)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
Caused by android.view.InflateException: Binary XML file line #11: Binary XML file line #11: Error inflating class fragment

Caused by android.view.InflateException: Binary XML file line #11: Error inflating class fragment

Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.wordpress.android.fluxc.model.SiteModel.isWPCom()' on a null object reference
       at org.wordpress.android.util.SiteUtils.isAccessibleViaWPComAPI(SiteUtils.java:37)
       at org.wordpress.android.ui.posts.adapters.PostsListAdapter.<init>(PostsListAdapter.java:109)
       at org.wordpress.android.ui.posts.PostsListFragment.getPostListAdapter(PostsListFragment.java:269)
       at org.wordpress.android.ui.posts.PostsListFragment.requestPosts(PostsListFragment.java:336)
       at org.wordpress.android.ui.posts.PostsListFragment.onCreateView(PostsListFragment.java:183)
       at android.app.Fragment.performCreateView(Fragment.java:2352)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:959)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1149)
       at android.app.FragmentManagerImpl.addFragment(FragmentManager.java:1255)
       at android.app.FragmentManagerImpl.onCreateView(FragmentManager.java:2315)
       at android.app.FragmentController.onCreateView(FragmentController.java:99)
       at android.app.Activity.onCreateView(Activity.java:6065)
       at android.support.v4.app.BaseFragmentActivityHoneycomb.onCreateView(BaseFragmentActivityHoneycomb.java:36)
       at android.support.v4.app.FragmentActivity.onCreateView(FragmentActivity.java:79)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:784)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:734)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:865)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:525)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:427)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:378)
       at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:288)
       at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
       at org.wordpress.android.ui.posts.PostsListActivity.onCreate(PostsListActivity.java:38)
       at android.app.Activity.performCreate(Activity.java:6912)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1126)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2900)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
```

**Comment by mzorz** [comment]
Closing - couldn't find it in latest versions

**Comment by maxme** [comment]
```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.PostsListActivity}: android.view.InflateException: Binary XML file line #11: Error inflating class fragment
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2118)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2145)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1211)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4847)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:535)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by android.view.InflateException: Binary XML file line #11: Error inflating class fragment
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
       at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:288)
       at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
       at org.wordpress.android.ui.posts.PostsListActivity.onCreate(PostsListActivity.java:38)
       at android.app.Activity.performCreate(Activity.java:5187)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1104)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2064)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2145)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1211)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4847)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:535)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NullPointerException
       at org.wordpress.android.util.SiteUtils.isAccessedViaWPComRest(SiteUtils.java:37)
       at org.wordpress.android.ui.posts.adapters.PostsListAdapter.<init>(PostsListAdapter.java:109)
       at org.wordpress.android.ui.posts.PostsListFragment.getPostListAdapter(PostsListFragment.java:269)
       at org.wordpress.android.ui.posts.PostsListFragment.requestPosts(PostsListFragment.java:336)
       at org.wordpress.android.ui.posts.PostsListFragment.onCreateView(PostsListFragment.java:183)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:807)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1013)
       at android.app.FragmentManagerImpl.addFragment(FragmentManager.java:1112)
       at android.app.Activity.onCreateView(Activity.java:4843)
       at android.support.v4.app.BaseFragmentActivityHoneycomb.onCreateView(BaseFragmentActivityHoneycomb.java:36)
       at android.support.v4.app.FragmentActivity.onCreateView(FragmentActivity.java:79)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:680)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
       at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:288)
       at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
       at org.wordpress.android.ui.posts.PostsListActivity.onCreate(PostsListActivity.java:38)
       at android.app.Activity.performCreate(Activity.java:5187)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1104)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2064)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2145)
       at android.app.ActivityThread.access$600(ActivityThread.java:130)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1211)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4847)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:535)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by maxme** [comment]
Similar:

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.posts.PostsListActivity}: android.view.InflateException: Binary XML file line #11: Binary XML file line #11: Error inflating class fragment
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2947)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
Caused by android.view.InflateException: Binary XML file line #11: Binary XML file line #11: Error inflating class fragment

Caused by android.view.InflateException: Binary XML file line #11: Error inflating class fragment

Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'int org.wordpress.android.fluxc.model.SiteModel.getOrigin()' on a null object reference
       at org.wordpress.android.util.SiteUtils.isAccessedViaWPComRest(SiteUtils.java:37)
       at org.wordpress.android.ui.posts.adapters.PostsListAdapter.<init>(PostsListAdapter.java:109)
       at org.wordpress.android.ui.posts.PostsListFragment.getPostListAdapter(PostsListFragment.java:269)
       at org.wordpress.android.ui.posts.PostsListFragment.requestPosts(PostsListFragment.java:336)
       at org.wordpress.android.ui.posts.PostsListFragment.onCreateView(PostsListFragment.java:183)
       at android.app.Fragment.performCreateView(Fragment.java:2352)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:959)
       at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1149)
       at android.app.FragmentManagerImpl.addFragment(FragmentManager.java:1255)
       at android.app.FragmentManagerImpl.onCreateView(FragmentManager.java:2315)
       at android.app.FragmentController.onCreateView(FragmentController.java:99)
       at android.app.Activity.onCreateView(Activity.java:6065)
       at android.support.v4.app.BaseFragmentActivityHoneycomb.onCreateView(BaseFragmentActivityHoneycomb.java:36)
       at android.support.v4.app.FragmentActivity.onCreateView(FragmentActivity.java:79)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:784)
       at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:734)
       at android.view.LayoutInflater.rInflate(LayoutInflater.java:865)
       at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:525)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:427)
       at android.view.LayoutInflater.inflate(LayoutInflater.java:378)
       at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:288)
       at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
       at org.wordpress.android.ui.posts.PostsListActivity.onCreate(PostsListActivity.java:38)
       at android.app.Activity.performCreate(Activity.java:6912)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1126)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2900)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3008)
       at android.app.ActivityThread.-wrap14(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1650)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6688)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
```

**Comment by mzorz** [comment]
Closed via #6007
